numba/tests/test_linalg.py

```diff
@@ -720,5 +781,117 @@ def check(a, **kwargs):
                                   (np.array([[1., 2., ], [np.inf, np.nan]],
                                             dtype=np.float64),))
 
+
+class TestLinalgQr(TestLinalgBase):
+    """
+    Tests for np.linalg.qr.
+    """
+
+    def sample_matrix(self, size, dtype, order):
```

Comment:
It *looks* like this is the same as `orth_fact_sample_matrix`, or I am overlooking something?